doc-tests/lib/travis.rb

```diff
@@ -0,0 +1,80 @@
+require "json"
```

Comment:
That introduces a dependency. At the moment, I only depend on ruby's standard library. Also, I require the use of the 'requests' API endpoint, which is not yet implemented in the library.